### DISCUSSION TITLE
Playback Events

### DISCUSSION THREAD

**Discussion by djbetterly** [artisan-roaster-scope/artisan#895] [General]
Is there a way to assign a quick button for enabling/disabling Playback Events?

**Comment by djbetterly** [comment]
Should I take that as a no?

**Comment by roasterdave** [comment]
Yes, you can create buttons to set or disable the playback mode.  There is an Artisan Command playbackmode(<int>) that sets playback mode to 0: off, 1: time, 2: BT, 3: ET.  Check the Help on the Config>> Events>> Buttons tab.

**Comment by MAKOMO** [comment]
There is also a keyboard shortcut to enable and disable quickly the playback of events. Pressing key j toggles this. The background profile name is set in red while playback is ON and black otherwise.
